apps/sms/js/threads.js

```diff
@@ -18,6 +18,10 @@
     this.messages = new Map();
   }
 
+  Thread.prototype.getDraft = function() {
```

Comment:
why not; I kind of like the name "hasDrafts" because it was really clear to read. But I agree the implementation was quite bad. I don't know if we can have the best of both worlds.

but again, maybe no need to think too much as this will need to be changed when we have the DB anyway.